modules/tasks/tasks/src/main/resources/webapp/partials/form.html

```diff
@@ -77,9 +77,61 @@ <h3 class="popover-title">{{msg('task.tooltip.availableTriggers')}}</h3>
                                     </ul>
                                     <div class="clearfix"></div>
                                 </div>
+                                <div>
+                                    <div ng-if="(task.trigger.subject != undefined)" class="col-md-10 col-sm-9">
```

Comment:
remove '!= undefined' and replace it with simply ng-if="task.trigger.subject". Goes for the entire file